### PR TITLE
vSphere component deprecation notices for RNs

### PR DESCRIPTION
**Pull request by codyhoag**
Previews:
* [Rel note entry](https://deploy-preview-40836--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-installation-vsphere-csi)
* [Deprecation table additions](https://deploy-preview-40836--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-deprecated-removed-features)

### CODE REVIEW

**Comment by codyhoag** [comment]
@jinyunma can you verify? Thanks!

**Comment by jinyunma** [comment]
LGTM
cc @duanwei33 from QE storage team to double confirm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -65,6 +65,17 @@ This release adds improvements related to the following components and concepts.
...
 Beginning with {product-title} {product-version}, if you configure a cluster with an existing IAM role, the installer no longer adds the `shared` tag to the role when deploying the cluster. This enhancement improves the installation process for organizations that want to use a custom IAM role, but whose security policies prevent the use of the `shared` tag.
 
+[id="ocp-4-10-installation-vsphere-csi"]
+==== CSI driver installation on vSphere clusters
+
+To install a CSI driver on a cluster running on vSphere, you must have the following components installed:
+
+* Virtual hardware version 15 or greater
+* vSphere version 6.7 Update 3 or greater
+* VMware eSXI version 6.7 Update 3 or greater
```
Comment (duanwei33) [inline release_notes/ocp-4-10-release-notes.adoc:75]: Hi, could we consider using the "ESXi" to be consistent? Thanks.

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**